review_app_status.py

```diff
@@ -225,6 +225,7 @@ def main() -> None:
             interval=args.interval,
         )
 
+    print(f"::set-output name=review_app_name::{reviewapp_build_data['environment']}")
```

Comment:
Follow the DRY (Don't repeat yourself) principle.
```python
reviewapp_name = reviewapp_build_data['environment'] # at https://github.com/niteoweb/reviewapps-deploy-status/pull/10/files#diff-3803057c7ddec8f39110437eb5f9b591R219

review_app_url = f"https://{reviewapp_name}.herokuapp.com"  # https://github.com/niteoweb/reviewapps-deploy-status/pull/10/files#diff-3803057c7ddec8f39110437eb5f9b591R220
```
```suggestion
    print(f"::set-output name=review_app_name::{reviewapp_name}")
```